ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java

```diff
@@ -24,9 +24,9 @@
 
 import kotlin.Unit;
 import kotlin.jvm.functions.Function2;
-import network.pocket.aion.AionContract;
-import network.pocket.aion.PocketAion;
-import network.pocket.aion.exceptions.AionContractException;
+import network.pocket.eth.EthContract;
+import network.pocket.eth.*;
+import network.pocket.eth.exceptions.EthContractException;
```

Comment:
Since you add pocket.eth.* the exceptions are already included, you can remove line 29